napari/utils/_dask_utils.py

```diff
@@ -4,8 +4,8 @@
 import contextlib
 from typing import Callable, ContextManager, Optional, Tuple
 
-import dask
 import dask.array as da
+import dask.config
```

Comment:
I think this doesn't help as the import log tree seems to indicate that there is effectively an `import dask` later due to this anyway. But maybe it could help in the future if dask internals are changed/optimized?